bin/do_deltas.py

```diff
@@ -84,6 +84,9 @@ def cont_fit(data):
     parser.add_argument('--nproc', type = int, default = None, required=False,
                     help = 'number of processors')
 
+    parser.add_argument('--veto-line-file',type = str,default=None,required=False,
```

Comment:
I'd call this option mask-file. It can be used to avoid portions wider than just a line (e.g. the entire LyB peak)